packages/atoms/src/NumPadButton.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { Pressable, PressableProps } from 'react-native';
+
+import { Text } from './Text';
+import { useNativeStyles, NativeStyleObject, prepareNativeStyle } from '@trezor/styles';
+
+export interface NumPadButtonProps extends Omit<PressableProps, 'style'> {
+    title: string;
+    onPress: () => void;
+    variant?: NumPadButtonVariant;
+    style?: NativeStyleObject;
+}
+
+export type NumPadButtonVariant = 'primary' | 'secondary' | 'default';
+
+export type NumPadButtonStyleProps = {
+    variant: NumPadButtonVariant;
+};
+
+const BUTTON_SIZE = 48;
+const BUTTON_FONT_SIZE = 34;
+
+export const numPadButtonStyle = prepareNativeStyle<NumPadButtonStyleProps>(
+    (utils, { variant }) => {
+        const numPadButtonColorSchemeStyles: Record<NumPadButtonVariant, NativeStyleObject> = {
+            primary: {
+                backgroundColor: utils.colors.green,
+            },
+            secondary: {
+                backgroundColor: utils.colors.gray300,
+            },
+            default: {
+                backgroundColor: 'transparent',
+            },
+        };
+
+        return {
+            width: BUTTON_SIZE,
+            height: BUTTON_SIZE,
+            alignItems: 'center',
+            justifyContent: 'center',
+            borderRadius: utils.borders.radii.round,
+            ...numPadButtonColorSchemeStyles[variant],
+        };
+    },
+);
+
+export const numPadButtonTextStyle = prepareNativeStyle<NumPadButtonStyleProps>(
+    (utils, { variant }) => {
+        const buttonTextStyles: Record<NumPadButtonVariant, NativeStyleObject> = {
```

Comment:
What about having it outside of the style? Would make the style more clear.

Reply:
It's easier to have then inside because you have access to theme utils